server.js

```diff
@@ -0,0 +1,72 @@
+var express = require("express");
+var app = express();
+var bodyparser = require("body-parser");
+var port = process.env.PORT || 3000; //listen to requests
+// on port 3000 - that is what is going on here
+// you can set port to something else but 3000 is standard
+
+var piglatinify = require("./lib/piglatinify.js");
+var basicYou = require("./lib/basicinfo.js");
+var quotes = require("./lib/quotes.js");
+var jokes = require("./lib/jokes.js");
+//var favorite = require("./lib/favorite.js");
+//var maybe = require("./lib/quotes.js");
+app.use(bodyparser.json());
+app.use(bodyparser.urlencoded({ extended: true }));
+
+//app.use(express.static(__dirname + "/app/"));
+
+/*var information	 = ["Do Better", "Try Harder", "You Got It", "Dont Stop Yet", "Awesome"];
+*/
+app.use(express.static(__dirname + "/app/"));
+
+var favorite = [
```

Comment:
next step would be to move this to another file - probably to the lib/favorite file you already have started.